### DISCUSSION TITLE
Numpy dependency without usage?

### DISCUSSION THREAD

**Discussion by gobo7793** [cwendt94/espn-api#227] [General] [answered]
Hi, I just wanted to update some pip packages in the Program I implemented some use cases of the espn_api package and wanted to update this to the current version 0.17.0. Now I got an error during updating the pip package because I don't have VC++ Build tools installed which numpy requires. Because I don't want to install the VC++ Build tools because I just don't need it for anything else and just to update espn_api it's too much.
So I searched where it's used and didn't find any usage of numpy (I searched for the term `numpy` in all files) in the current version of the master branch of this repo. I didn't analyzed the whole code. So my question: Why is it still there? Did I missed something?

**Comment by cwendt94** [comment, marked as answer]
The numpy package is used by `pandas` in two of the hockey classes (Player and Team). However the goal is to take out math and analytic specific logic and leave that up to the user of the package and not require it. This was added in the first implementation of hockey but the data can be derived outside of the package.

I want to keep this package light and not have a lot of dependencies. Thanks for bringing this up, I think its a good time to remove the `numpy` and `pandas` dependencies.

**Reply by cwendt94** [reply]
The dependencies have been removed in v0.18.0
